### PR TITLE
Finalize artifacts and targets

### PR DESCRIPTION
**Pull request by idavis**
- New sdist for platforms we don't release
- Creating qir stdlib releases
- Simplifying CI now that releases are done through ADO
- Removing aarch mac and universal builds as they don't fully work xplat

### CODE REVIEW

File: notice.ps1
Comment (swernli) [inline notice.ps1:1]: What does the empty notice.ps1 at the root do? Is that needed for packaging or leftover from an attempt at something?

**Review by swernli — APPROVED** [review]
One question on an empty file, but otherwise things look good!

File: notice.ps1
Comment (idavis) [reply notice.ps1:1]: GH can't render metadata changes. This file is now executable.

**Review by swernli — APPROVED** [review]
Good catch on removing the unused feature!

**Pull request merged**